README.md

```diff
@@ -0,0 +1,48 @@
+# Guitar API
+
+## Table of Contents
+- [Description](#description) :page_with_curl:
+- [Requirements](#requirements) :hammer_and_wrench:
+- [Getting Started](#getting-started)
+   - [Installation](#installation)
+- [Documentation](#official-documentation)
+- [Feature Roadmap](#feature-roadmap)
+
+## Description 
+The Guitar API is a RESTful API that provides information about various guitar makes and modes, along with information about famous mucisians who have used those models at some point in their careers. This API allows developers to access ruitar-related information to build applications, websites, or any other projects that require guitar-related data.
+
+## Requirements
+- Go Programming Language
+- PostgreSQL
+   - For storing and retrieving API database information
+- pgAdmin
+- API Testing Tool
+   - [Postman][https://www.postman.com/] or something similar
+
+## Getting Started
+In it's current version, the API requires the above tools to be installed locally on one's machine. Continue below for installation docs to install each of the tools on your operating system.
+
+### Installation
```

Comment:
```suggestion
### Setup
```

A bit pedantic, but we’re setting this repo up locally, and not really “installing” per-se